src/kibana/directives/saved_object_finder.js

```diff
@@ -104,100 +105,125 @@ define(function (require) {
           filterResults();
         });
 
-        $scope.selectedItem = false;
-        $input.on('keydown', (function () {
-          var enter = 13;
-          var up = 38;
-          var down = 40;
-          var left = 37;
-          var right = 39;
-          var esc = 27;
-
-          var scrollIntoView = function ($el, snapTop) {
-            var el = $el[0];
-
-            if (!el) return;
-
-            if ('scrollIntoViewIfNeeded' in el) {
-              el.scrollIntoViewIfNeeded(snapTop);
-            } else if ('scrollIntoView' in el) {
-              el.scrollIntoView(snapTop);
-            }
-          };
+        //manages the state of the keyboard selector
+        self.selector = {
+          enabled: false,
+          index: -1
+        };
 
-          return function (event) {
-            var $next;
-            var goingUp;
+        //key handler for the filter text box
+        self.filterKeyDown = function ($event) {
+          if (keymap[$event.keyCode] !== 'tab')
+            return;
```

Comment:
Curly braces are required if you're doing multi-line conditionals. Here, return should just be on the same line.